common/app/views/fragments/amp/stylesheets/buttons.scala.html

```diff
@@ -48,6 +48,7 @@
     content: "";
     background-color: #eaeaea;
     border-radius: 1.125rem;
+    -webkit-border-radius: 1.125rem;
```

Comment:
As far as I can tell we do that while compiling our sass (mentioned in our gulp and grunt files).

If that is the case that doesn't work for AMP.